Tests/Testably.Abstractions.Tests/FileSystem/Directory/GetFilesTests.cs

```diff
@@ -10,6 +10,24 @@ public abstract partial class GetFilesTests<TFileSystem>
 	: FileSystemTestBase<TFileSystem>
 	where TFileSystem : IFileSystem
 {
+	[SkippableTheory]
+	[AutoData]
+	public void GetFiles_WithRelativePathAndSubfolders_ShouldReturnRelativeFilePath(string subfolder1, string subfolder2, string[] files)
+	{
+		string workingDirectory = System.IO.Path.Combine(BasePath, subfolder1, subfolder2);
+		FileSystem.Directory.CreateDirectory(workingDirectory);
+		foreach (string file in files)
+		{
+			FileSystem.File.Create(System.IO.Path.Combine(workingDirectory, file));
+		}
+		
+		FileSystem.Directory.SetCurrentDirectory($@".\{subfolder1}");
```

Comment:
You have to use the correct [`Path.DirectorySeparatorChar`](https://learn.microsoft.com/en-us/dotnet/api/system.io.path.directoryseparatorchar), so that the tests also succeed on Linux and MacOS